integration_tests/features/steps/challenge_spent_coin_flow.py

```diff
@@ -0,0 +1,128 @@
+import time
+
+from behave import given, then, when
+from web3.auto import w3
+
+from plasma_cash.dependency_config import container
+
+userA = '0xb83e232458A092696bE9717045d9A605FB0FEc2b'
```

Comment:
I think we can start to re-use all these constants and the util functions: `address_equals` & `has_value`